public/javascript/projects.js

```diff
@@ -10,6 +10,10 @@
 
     buttonEls$.click(function() {
       var projectName = this.dataset.projectName;
+      
+      var confirm = window.confirm('Do you really want to delete project ' + projectName + '? This cannot be undone!')
+      if(!confirm) return;
```

Comment:
Can we always use brackets even for one line conditionals? It's a JS standard and I think the linter will complain.